apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx

```diff
@@ -816,6 +822,39 @@ const PoolCandidatesTable = ({
           ),
       },
     ),
+    columnHelper.accessor(
+      ({ poolCandidate: { assessmentStep } }) => assessmentStep,
+      {
+        id: "assessmentStep",
+        header: intl.formatMessage(tableMessages.assessmentStep),
+        cell: ({
+          row: {
+            original: {
+              poolCandidate: {
+                assessmentStep,
+                pool: { assessmentSteps },
+              },
+            },
+          },
+        }) => {
+          const step = assessmentSteps?.find(
+            (s) => s?.sortOrder === assessmentStep,
+          );
+          return step?.title?.localized
+            ? intl.formatMessage(
+                {
+                  defaultMessage: "Step {step}",
+                  id: "+O4pHs",
+                  description: "Label for current assessment step.",
+                },
+                { step: assessmentStep },
+              ) +
+                intl.formatMessage(commonMessages.dividingColon) +
+                step.title.localized
+            : "";
+        },
+      },
```

Comment:
We've got quite a lot of similar looking strings and it can be kinda hard to tell if they are being generated the same way or not. 

<img width="446" height="386" alt="image" src="https://github.com/user-attachments/assets/ba58a8ea-9aa6-4be5-b477-b5a8d2f7ee60" />

Looking at `MoreActions.tsx` I see what seems to be the exact same format being generated slightly differently. 

<img width="831" height="633" alt="image" src="https://github.com/user-attachments/assets/0f1dd9ba-d92e-4b65-b78e-dbc7df260bca" />

Can we get them synced up and maybe more reusable?